test/button-state.test.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
```

Comment:
Do you think pure javascript should be tested in WCT or just web components?

cc @nfiniteset